opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -734,28 +736,29 @@ namespace Opm {
 
 
 
-    template<typename FluidSystem, typename BlackoilIndices, typename ElementContext, typename MaterialLaw>
-    template<typename FluidState>
+    template<typename TypeTag>
+    template<typename intensiveQuants>
```

Comment:
capital `I`, please. (also, you don't really need the template parameter if you have the TypeTag: you can use `typename GET_PROP_TYPE(TypeTag, IntensiveQuantities)`. Note that for methods directly specified in the class body the `IntensiveQuantities` could be typedef'd once for the whole class, so this would become a plain `IntensiveQuantities`.)